csuntechlab/iSTART
# STR-243 -- Rewire module progress

# Description
Wired module progress for `get-module-progress` & `set-module-progress`. 
Added progress checking and initial load icon.
Upgraded last slide checking to send _User_ & _Admin_ emails.
  
# Testing
1. Login as a participant.
2. Check for no current module.
3. Check save progress.
4. Check for email on last slide.
